sirepo/template/hdf5_util.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""HDF5 utilities.
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdp
+from sirepo.template import template_common
+import numpy as np
+
+
+class HDF5Util:
+    # TODO(rorour): add docs for heatmap & lineplot
+    def __init__(self):
+        pass
```

Comment:
Here it should take the hdf5 file in the constructor, and then when plotting, the file should be opened with h5py and the file handled passed to the callbacks if needed.

Reply:
d27e19ab27a41125f8de5d4f719ec9b485904168